src/storable/tuples.rs

```diff
@@ -186,16 +172,14 @@ fn decode_size_lengths(mut encoded_bytes_to_store: u8, number_of_encoded_lengths
 // The element is assumed to be at the beginning of `dst`.
 // Returns the number of bytes written to `dst`.
 fn encode_tuple_element<T: Storable>(dst: &mut [u8], bytes: &[u8], last: bool) -> usize {
-    let mut bytes_written: usize = 0;
-    let size = bytes.len();
-
+    let mut written = 0;
```

Comment:
nit: should be called `offset` for consistency with the code below?

Reply:
done.